CONTRIBUTING.md

```diff
@@ -15,38 +15,44 @@ Can you read this? Then you! Simply consult the short [style guidelines](#style-
-4. `git add` the change(s)
-5. `git commit` with a useful commit message!
-6. `git push`
-7. Make a pull-request (shiny green button on your fork of the repo's GitHub webpage.)
+1. If contributing to gh-pages:
+   * `git checkout gh-pages`
+1. Change something (_after_ consulting the [style guidelines](#style-guidelines))
+1. `git add` the change(s)
+1. `git commit` with a useful commit message!
+1. `git push`
+1. Make a pull-request (shiny green button on your fork of the repo's GitHub webpage.)
 
 Alternatively, you may send your contributions via e-mail attachment to HPC-Help@nrel.gov with subject of "NREL HPC GitHub Contribution" and the body detailing what changes you made (the more specific the better).
 
 ### Why should I contribute?
-Something something good Samaritan, benefit the community, searchable knowledge-base, etc. 
+Benefit the community, searchable knowledge-base, etc. 
 
 ---
 
 ## Style Guidelines
 
->### **TL;DR&mdash;this repository should be predominantly composed of scripts/source code/executables or other things that _do_ something on the HPC systems, and the Wiki should predominantly be for explanations about what/how/why.**
->### That said, still getting familiar with the details below will improve the chance your contributions are submitted in a a readily-acceptable format.
-
 ### ***New to Markdown?***
-Appropriately enough, we have a documentation for that! Simply start with the [README in the Markdown module](/markdown/README.md). Not to mention, the raw-contents of any `.md` file can be used as a reference for how to style content a certain way.
+Appropriately enough, we have a documentation for that! Simply start with the [README in the Markdown module](/general/beginner/markdown/README.md). Not to mention, the raw-contents of any `.md` file can be used as a reference for how to style content a certain way.
+
+### Markdown content and structure
+* The intent of training or tutorial materials is to show someone who does not know how to do what you do how to do it. Err on the side of verbosity.
 
-Note that git (the version control protocol) is not equivalent to GitHub (the git repository hosting web service, which implements the git protocol). This is an important distinction to be sure of before proceeding. There are other git-hosting services such as GitLab which function similarly to GitHub, but both use the git protocol. These hosting-services provide different decorative features to make the repositories more visually pleasing/intuitive to interact with.
+* Where you can assume that a user should have worked through training materials conceptually prior to the current ones, cross-reference them (mention and hyperlink). This also applies to information on the HPC website. In general, where you can refer someone to well crafted information online that's more extensive or well constructed than you have time or space to write, do so through a hyperlink.
```

Comment:
Where you can assume that a user should have worked through other training materials or concepts prior to your contribution, please cross-reference the other material (with a mention and a hyperlink). This also applies to information on the HPC website. In general, where you can refer someone to well-crafted information online that's more extensive or well-constructed than you have time or space to write, do so through a hyperlink.